jetson-xavier.coffee

```diff
@@ -2,9 +2,8 @@ deviceTypesCommon = require '@resin.io/device-types/common'
 { networkOptions, commonImg, instructions } = deviceTypesCommon
 
 BOARD_PREPARE  = 'Put the NVidia Jetson Xavier board in recovery mode'
-FLASH_SCRIPT_DOWNLOAD = 'Obtain the flashing script from: https://github.com/balena-os/balena-jetson-nano/blob/master/layers/meta-balena-jetson/recipes-scripts/flash_xavier.sh'
-FLASH_SCRIPT = 'Unzip BalenaOS image and execute: sudo ./flash_xavier.sh <path_to_balenaOS.img>'
-DONE_FLASHING  = 'Follow the instructions printed by the flashing script'
+FLASH_TOOL = 'Unzip BalenaOS image and use the Jetson Flash tool to flash the board'
```

Comment:
Let's also put a link here to where the flashing tool is.

Reply:
Thanks! Sure, will update